### ISSUE TITLE
Episode 3 (Starting with data) text error

### ISSUE THREAD

**Issue by agully1** [datacarpentry/r-socialsci#349]
In Episode 3 of this lesson, Starting with Data ([link](https://datacarpentry.org/r-socialsci/02-starting-with-data/index.html)), in the final section "Formatting Dates", the text includes "As a reminder from earlier in this lesson, the best practice for dealing with date data is..."

This has not been mentioned anywhere earlier in the lesson as far as I can see. Consider changing this text?

**Comment by juanfung** [comment]
@agully1 Good catch. Would you be interested in submitting a PR for this? (No need for multiple PRs, esp since it's the same episode, you can submit one PR for both issues)
